source

```diff
@@ -103264,19 +103278,23 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>data state</span>. Emit the current DOCTYPE token.</dd>
 
    <dt>U+0022 QUOTATION MARK (&quot;)</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's system identifier to the empty string (not
+   <dd>This is a <dfn data-x="parse-error-missing-whitespace-after-doctype-public-identifier">missing-whitespace-after-doctype-public-identifier</dfn>
```

Comment:
Should be same as below: `missing-whitespace-between-doctype-public-and-system-identifiers`. I guess it's caused by my misleading comment again.